cstore_fdw.c

```diff
@@ -251,6 +233,13 @@ CStoreProcessUtility(Node *parseTree, const char *queryString,
 
 			DeleteCStoreTableFiles(fileName);
 		}
+
+	}
+	else if (nodeTag(parseTree) == T_TruncateStmt)
+	{
+		TruncateCStoreTable((TruncateStmt*) parseTree);
+		CallPreviousProcessUtility(parseTree, queryString, context,
+								   paramListInfo, destReceiver, completionTag);
```

Comment:
1. TruncateCStoreTable -> TruncateCStoreTables?
2. I was confused here for a while that why the standard utility is not throwing error here, and that was because I couldn't tell from the name "TruncateCStoreTable" that it removes cstore tables from parseTree->relations. Maybe we should break TruncateCStoreTable into 2 or more functions to make it more explicit.
3. What happens when we do ```TRUNCATE cstore_table, regular_table, file_fdw_table```? I guess we truncate the cstore table, then standard truncate errors out and doesn't truncate any of the other tables?
4. ```(TruncateStmt*) parseTree``` -> ```(TruncateStmt *) parseTree```? (added a space character)

Given above, what about something like this?
```C
	else if (nodeTag(parseTree) == T_TruncateStmt)
	{
		TruncateStmt *truncateStatement = (TruncateStmt *) parseTree;
		List *cstoreTableList = FindCStoreTables(truncateStatement->relations);

		/*
		 * Truncate non-cstore tables. If an error happens here, we won't continue
		 * with cstore tables.
		 */
		truncateStatement->relations = list_difference(truncateStatement->relations,
													   cstoreTableList);
		CallPreviousProcessUtility(parseTree, queryString, context,
								   paramListInfo, destReceiver, completionTag);

		/* truncate cstore tables */
		TruncateCStoreTables(cstoreTableList);
	}
```